### PR TITLE
Preparations for CS2 changes

### PR DESCRIPTION
**Pull request by pyrello**
I think this PR will make finishing #4091 easier.

# How to test

I tested all the split definitions by re-exporting them and making any changes necessary to prevent substantive changes from occurring.

Review site split file changes (not including the split configs themselves) and test as necessary.

### CODE REVIEW

File: config/sites/icsa.uiowa.edu/config_split.config_split.site.yml [hunk 1]
```diff
@@ -25,6 +19,9 @@ graylist:
   - system.menu.services
   - views.view.projects
   - views.view.reports
+  - 'field.storage.node.field_project_*'
+graylist:
+  - core.date_format.medium_date
```
Comment (joewhitsitt) [inline config/sites/icsa.uiowa.edu/config_split.config_split.site.yml]: This is a complete split as medium_date is custom and not part of default

**Review by joewhitsitt — APPROVED** [review]
I have reviewed each file. With your statement that things like `core.entity_*_display` and `core.base_field_override.*` are auto included based on the entity type being captured then it looks pretty good

**Comment by joewhitsitt** [comment]
If you aren't in a rush, I could do a `blt dsa` on sites affected by this to see if there are issues.

File: config/sites/icsa.uiowa.edu/config_split.config_split.site.yml (hunk 1 above)
Comment (pyrello) [reply config/sites/icsa.uiowa.edu/config_split.config_split.site.yml]: Forgot to push up this change. D'oh!

**Pull request merged**